backend/src/main/java/com/sep6/backend/repository/AccountsRepositoryImpl.java

```diff
@@ -30,13 +30,14 @@ public Optional<Account> editAccount(int id, Account account)
         try {
             Account edited = jpaRepository.findById(id).orElseThrow();
 
+            edited.setUsername(account.getUsername());
+            edited.setPassword(account.getPassword());
             edited.setEmail(account.getEmail());
             edited.setName(account.getName());
             edited.setCountry(account.getCountry());
             edited.setProfilePictureUrl(account.getProfilePictureUrl());
-            edited.setPassword(account.getPassword());
-            edited.setEnabled(account.isEnabled());
-            edited.setFavourites(account.getFavourites());
+            edited.setDateOfBirth(account.getDateOfBirth());
+            edited.setGender(account.getGender());
```

Comment:
Dont be too open minded Tomas :D We shouldnt allow gender change :D 
And I think you cannot really change you birthdate

Reply:
True, I did not realise the meaning of fields, I just saw fields of an object 😂😂😂

Reply:
I mean if you can change your gender, you can also change your birthday

Reply:
We may have customers that complain cuz they cannot change the gender 😆😆